src/Developer/Encryption/FieldLevelEncryption.php

```diff
@@ -12,8 +12,6 @@
  */
 class FieldLevelEncryption {
 
-    private function __construct() {}
```

Comment:
I would keep private constructors, it makes `FieldLevelEncryption` looks like a static class and prevents client from trying to instantiate it.

Maybe use:

```php
private function __construct() {
  // This class can't be instantiated
}
```
?